include/swift/Strings.h

```diff
@@ -16,72 +16,72 @@
 namespace swift {
 
 /// The extension for serialized modules.
-static const char SERIALIZED_MODULE_EXTENSION[] = "swiftmodule";
+constexpr const char SERIALIZED_MODULE_EXTENSION[] = "swiftmodule";
```

Comment:
These should still be `static`, thanks! Otherwise they'll get emitted into every file that includes this one.